src/presign/participant.rs

```diff
@@ -50,7 +51,22 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use tracing::{error, info, instrument};
 
-use super::round_two;
+/// Storage identifiers for the presign protocol.
+#[derive(Clone, Copy, Debug, Serialize)]
+#[serde(tag = "Presign")]
+pub(crate) enum StorageType {
```

Comment:
Any preference on naming this `PresignType` instead of `StorageType`?

Reply:
I prefer not duplicating the module name in the type name, although I agree `StorageType` is not a great name but I guess it'll go away soon enough :-)

Reply:
Sounds good!